app/Libraries/Payments/PaypalPaymentProcessor.php

```diff
@@ -70,16 +70,16 @@ public function getNotificationType(): string
         static $rejectedStatuses = ['Declined', 'Denied', 'Expired', 'Failed', 'Voided'];
 
         $status = $this->getNotificationTypeRaw();
-        if (in_array($status, $paymentStatuses, true)) {
+        if ($this->shouldIgnore($status)) {
```

Comment:
probably can use a note on why this needs to be first (or maybe just inline the function)